xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/UserWikiSheet.xml

```diff
@@ -468,4 +415,201 @@ return XWiki;
+      #else
+        #set($temp = $xwiki.ssx.use('WikiManager.UserWikiSheet'))
+        #set($temp = $xwiki.jsx.use('WikiManager.UserWikiSheet'))
+        #set($temp = $doc.use($obj))
+        #set ($currentUser = $xcontext.user)
+        #if (!$currentUser.startsWith("${services.wiki.mainWikiId}:"))
+          #set ($currentUser = "${services.wiki.mainWikiId}:${currentUser}")
+        #end
+        #set($documentUser = $doc.prefixedFullName)
+        #set($isMyProfile = ($currentUser == $documentUser))
+      {{html clean='false'}}
+      &lt;div class='half column'&gt;
+        &lt;div class='profile-section highlighted-profile-section'&gt;
+          &lt;h1&gt;$services.localization.render('platform.wiki.users.profile.joinedwikis.label')&lt;/h1&gt;
+            #set ($joinedWikis = [])
+            #set ($wikis = $services.wiki.getAll())
+            #foreach ($wiki in $wikis)
+            #if($services.wiki.user.isMember($documentUser, $wiki.id))
+              #set($temp = $joinedWikis.add($wiki))
+            #end
+            #end
+            #if ($joinedWikis.size() == 0)
+        &lt;span&gt;#if ($isMyProfile)$services.localization.render('platform.wiki.users.profile.joinedwikis.notMemberOfAnyWiki')#{else}$services.localization.render('platform.wiki.users.profile.joinedwikis.userNotMemberOfAnyWiki')#end&lt;/span&gt;
+      #else
+        #set ($discard = $xwiki.jsx.use('XWiki.Notifications.Code.Macro.NotificationsMacro'))
+        &lt;ul id='wikis'&gt;
+        #foreach ($wiki in $joinedWikis)
+          &lt;li id='$wiki.id'&gt;
+            &lt;span class='wikiHomepage'&gt;&lt;a href='$xwiki.getURL($wiki.mainPageReference)'&gt;$wiki.prettyName&lt;/a&gt;&lt;/span&gt;
+            &lt;span class='wikiActivity'&gt;&lt;a href='$doc.getURL('view', "category=wikis&amp;amp;wikiName=$wiki.id")'&gt;$services.localization.render('platform.wiki.users.profile.activity.show') &amp;#187;&lt;/a&gt;&lt;/span&gt;
+          &lt;/li&gt;
+        #end
+        &lt;/ul&gt;
+      #end
+        &lt;/div&gt;
+      &lt;/div&gt;
+      {{/html}}
+
+      {{html clean='false' wiki='true'}}
+      &lt;div id='wikiActivity' class='half column'&gt;
+        &lt;div id='wikiActivityColumn'&gt;
+          #if("$!request.wikiName" != '')
+            {{include reference="WikiManager.WikiActivityServiceCode" /}}
+          #end
+        &lt;/div&gt;
+      &lt;/div&gt;
+      &lt;div class="clearfloats"&gt;&amp;nbsp;&lt;/div&gt;
+      {{/html}}
+      #end## User object exists
+      {{/velocity}}</content>
+    </property>
+    <property>
+      <extensionPointId>org.xwiki.plaftorm.user.profile.menu</extensionPointId>
+    </property>
+    <property>
+      <name>org.xwiki.platform.wiki.ui.mainwiki.menu</name>
+    </property>
+    <property>
+      <parameters>id=wikis
+name=$services.localization.render('platform.wiki.menu.userwikis')
+icon=list
+# isActive: The current wiki is the main wiki.
+isActive=#if($xcontext.isMainWiki())true#{else}false#end
```

Comment:
wouldn't `isActive=$context.isMainWiki()` work?

Reply:
It works but makes less explicit what the intended output is.

Reply:
Well it's expected that `isMainWiki()` returns a boolean value, and it's easy to check it in the javadoc. Personally I find it cleaner. Moreover we should understand it with the comment above (which I don't find very clear btw :) you don't explain why it's active only if the current wiki is the main wiki).

Now it's really a small detail.